packages/rum-recorder/src/domain/rrweb-snapshot/types.ts

```diff
@@ -0,0 +1,97 @@
+export enum NodeType {
+  Document,
+  DocumentType,
+  Element,
+  Text,
+  CDATA,
+  Comment,
+}
+
+export type documentNode = {
```

Comment:
I find types starting with a lowercase letter to be kind of confusing. I changed the casing in `rrweb/types.ts`, could you do the same here?